site/docs/v1/tech/lambdas/nintendo-reconcile.adoc

```diff
@@ -42,7 +37,7 @@ Navigate to [breadcrumb]#Settings -> Identity Providers# and select your existin
 
 == Example lambda
 
-There is not a example Nintendo reconcile lambda provided by FusionAuth. The following is an example lambda you can use as a starting point.
+There is no example Nintendo reconcile lambda provided by FusionAuth. The following is an example lambda you can use as a starting point.
```

Comment:
We have other IdPs that don't ship with a default lambda, such as Epic and Steam. We should either follow those examples, or you could also add this type of example to other IdPs that do not ship a default reconcile lambda.

Reply:
Yeah, that was already there so I just left it. I'll remove it to match the others.

Reply:
Removed.